src/main/java/calculator/Operator.java

```diff
@@ -0,0 +1,46 @@
+package calculator;
+
+public enum Operator {
+
+    ADD("+") {
+        @Override
+        public int apply(int a, int b) {
+            return Calculator.add(a, b);
```

Comment:
```suggestion
            return a + b;
```
이 정도 구현이면 직접 구현해도 되지 않을까?